torchtune/modules/_export/install_requirements.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+NIGHTLY_VERSION="dev20241121"
+
+# Install pytorch nightly for export-friendly modules to run.
+pip install torch==2.6.0.${NIGHTLY_VERSION} torchvision==0.20.0.${NIGHTLY_VERSION} --extra-index-url https://download.pytorch.org/whl/nightly/cpu
```

Comment:
This sh file doesn't really make sense to me. The goal of the export folder is to house export friendly modules not modify the library setup. Currently torchtune operates under the assumption that the user installs pytorch themselves and some of our features our gated behind nightlies where we'll tell the user to upgrade to nightlies if they want to use that feature. The same pattern should be followed here.